src/mono/sample/wasm/browser-advanced/Wasm.Advanced.Sample.csproj

```diff
@@ -4,6 +4,9 @@
     <_SampleProject>Wasm.Advanced.Sample.csproj</_SampleProject>
     <EnableAggressiveTrimming>true</EnableAggressiveTrimming>
     <PublishTrimmed>true</PublishTrimmed>
+    <!-- add OpenGL emulation -->
+    <EmccExtraExportedRuntimeMethods>'GL'</EmccExtraExportedRuntimeMethods>
```

Comment:
Is this always a bunch of strings? Maybe it is more MSBuild normal to do a semicolon list of things - and then quote and comma in the targets file.

Reply:
Item group in radical 's comment is better